drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -3,6 +3,11 @@
 #include "os/os.h"
 #include "rasterizer_canvas_gles3.h"
 
+#ifdef IPHONE_ENABLED
+// for some reason glClearDepth seems to have been removed in iOS ES3.h
+#define glClearDepth glClearDepthf
```

Comment:
I'll change it if you're sure, I think its neater, however according to this officially it was introduced in 4.1:
https://www.opengl.org/sdk/docs/man/html/glClearDepth.xhtml